package.json

```diff
@@ -4,10 +4,11 @@
   "description": "Nitro's Design System",
   "main": "./app/pb_kits/playbook/packs/index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "./bin/webpack --profile --progress",
```

Comment:
Pretty standard across our web apps to have this here. It is nice to have a shortcut when you want to test out webpack without running `make start` and waiting forever.